end-to-end/src/test/java/nl/tudelft/wdm/group1/endToEnd/EndToEndTest.java

```diff
@@ -39,6 +41,9 @@ public void setUp() throws MalformedURLException {
         }
 
         RestAssured.baseURI = baseURL.getProtocol() + "://" + baseURL.getHost();
+
+        Awaitility.setDefaultPollInterval(500, TimeUnit.MILLISECONDS);
+        Awaitility.setDefaultPollDelay(100, TimeUnit.MILLISECONDS);
```

Comment:
Is this config automatically used for all tests? Because I don't see something similar for the non-e2e tests.